scripts/simtest/config-patch

```diff
@@ -1,21 +1,24 @@
-diff --git a/.cargo/config b/.cargo/config
```

Comment:
How are simtests working if the path to config is wrong?

Reply:
They don't use this file

Reply:
I'm not sure to follow. This section is adding `msim` to the `rustflags` so I'm assuming it's for the simtests.

Reply:
No, the simtests use `cargo-simtest` and build the command with the build flags in it

Reply:
So what was relying on this section of the patch? Actually nothing?